### ISSUE TITLE
Feature request: bring back projects API support

### ISSUE THREAD

**Issue by toinbis** [ZenHubIO/support#1006]
TL;DR: I've been looking for a product exactly as zenhub. Found it, was excited, used it for 5 mins, missed project support, googled https://www.zenhub.com/blog/dispatches-from-github-universe/ and realised zenhub does not support our workflow due to the lack of project API support.

Extensive version:
- We searched for a tool, which would work for developers team as well as content team, SEO team, management team, design team, ops team, etc. etc.
- We created respective teams(i.e. "SEO team", "Design team") and single repository for all of the projects/tasks of that team, i.e. ("SEO team tasks&projects", "Design team tasks&projects repo"). Each of the repo contains several projects, i.e. "SEO team tasks&projects" repository has "onsite", "offsite", "outreach", "reporting" projects.
- Filtering by project is the main feature needed for proper task delegation and progress tracking for project managers. Sadly, ZenHub does not support that.

I understand ZenHub is primarily a tool for developers to smoothen their work flow. But projects feature by github makes github issues/labels/projects/milestones/teams ecosystem as a proper api for any https://en.wikipedia.org/wiki/Getting_Things_Done workflow. Github made it available, you decided to skip supporting that. I imagine that initial feedback you got from devs were "we don't need that". But that essentially means "we do not use github projects at all", cause for majority of github repos, I'd guess, the projects feature is not used at all. That's an obvious answer from them. But what about the whole huge market of companies, which has many development and non-development tasks/activities/projects, which would be very happy to use github+zenhub as a single project/task tracking software to replace asana/trello, which are not suitable for development tasks setup? Huge market share opportunity missed, way bigger than the current, developer-only. And the "combined with the resources necessary to support the Projects API, have led to our decision to remove the integration" argument, just, well, sorry, I don't buy that :) Product built to smoothen work with github decides to... skip some of the github features, because, well, they need to support that. Both arguments why you decided to cease support for project API are actually, imho, invalid. Sorry to say that :)

Tried ZenHub as a first recommendation after slack in https://github.com/integrations/feature/project-management , but now how to keep going down the list, though initial experience with ZenHub was just freaking good!

One last question - maybe you could re-enable the feature with a "alpha feature" selection and allow to use it only for those how are fine by alpha stage for the feature? We'd be OK to pay extra for the "projects" functionality, am sure quite some other teams out there would be ready to do that. Just in case you'd need to re-evaluate this decision of ceasing projects api support, I'd be glad to share the experience on how I managed to make ZenHub work for both developers and non-developers team. 

Why u don't want those bussiness people on-board, who are always more keen to open their purses and give you cash? :)

Sad and a bit angry,
Tomas:)

**Comment by toinbis** [comment]
Update: discovered the "epics" feature, maybe that'll work as a (temporary) workaround. Off to try that out :)

**Comment by kendramoroz** [comment]
Hey @toinbis Kendra here from the ZenHub success team! Thanks for the great description of how your team works and the process you took to get to ZenHub :). We're happy to have you here! I'd love to schedule a time to walkthrough ZenHub and go through how we can get you and your team setup, answering any questions you have along the way! Let me know the best way to reach you to dig deeper and or send me an email at kendra@zenhub.com 🚀

**Comment by kendramoroz** [comment]
@toinbis Hello again! How are epics working for you so far? Anything I can help with here? 🌷
